FIX_GOOGLE_CALENDAR.md

```
@@ -0,0 +1,216 @@
+APP_URL=https://webapp-frtjapec0-ikips-projects.vercel.app
+
+# Base de données
+DATABASE_URL=postgresql://...
+```
+
+### ⚠️ Important pour la clé privée
+
+1. **Garder les guillemets doubles** autour de la clé complète
+2. **Les `\n` doivent être littéraux** (pas de vrais sauts de ligne dans Vercel UI)
+3. Format attendu dans Vercel :
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)